test/edu/stanford/dlss/was/TestWasapiDownloader.java

```diff
@@ -9,18 +11,45 @@
 
 public class TestWasapiDownloader {
 
-    @Before
```

Comment:
I like the 4 indents that were already present in this file - is there a particular reason to change it?